requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
+# This file is autogenerated by pip-compile
```

Comment:
Are you on the 'right' versions of python & pip? These changes seem a bit sus

Reply:
ah annoying, yeah i'm on 3.9. Have reverted all the other changes.